rest-api/tools/import_organizations.py

```diff
@@ -173,10 +174,20 @@ def _entity_from_row(self, row):
                 city=city,
                 state=state,
                 zipCode=zip_code,
+                latitude=latitude,
+                longitude=longitude,
                 phoneNumber=phone,
                 adminEmails=admin_email_addresses,
                 link=link)
 
+  def _get_lat_long_for_site(self, address_1, city, state):
+    api_key = os.environ.get('API_KEY')
+    gmaps = googlemaps.Client(key=api_key)
+    geocode_result = gmaps.geocode(address_1 + '' +  city + ' ' +  state)[0]
```

Comment:
Presumably this could return an error / something saying that the geocode failed?

We should handle that gracefully -- log a warning and proceed with no lat/lng.